Ginger/Ginger/Environments/AddNewDatabasePage.xaml.cs

```diff
@@ -0,0 +1,345 @@
+﻿using Amdocs.Ginger.Common;
+using Ginger.BusinessFlowPages.ListHelpers;
+using GingerCore;
+using GingerCore.Environments;
+using GingerCore.GeneralLib;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Controls.Primitives;
+using static GingerCore.Environments.Database;
+
+namespace Ginger.Environments
+{
+    /// <summary>
+    /// Interaction logic for AddNewDatabasePage.xaml
+    /// </summary>
+    public partial class AddNewDatabasePage : Page
+    {
+        GenericWindow _pageGenericWin = null;
+        private readonly ObservableList<IDatabase> dataSourceList;
+        private readonly Database database;
+        private readonly Button testConnectionButton = new();
+        private readonly Button okBtn = new ();
+        private readonly DatabaseListViewHelper dbListViewHelper;
+        public AddNewDatabasePage(ObservableList<IDatabase> dataSourceList, Context context, DatabaseListViewHelper dbListViewHelper)
+        {
+            this.dbListViewHelper = dbListViewHelper;
+            this.dataSourceList = dataSourceList;
+            InitializeComponent();
+            xDatabaseComboBox.ItemsSource = GingerCore.General
```

Comment:
Use the same function SQL_Selected to avoid code duplication to load DB types in drop down